src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1810,10 +1810,10 @@ int shim::xclCloseContext(const uuid_t xclbinId, unsigned int ipIndex)
 
     if (ipIndex < mCuMaps.size()) {
 	    // Make sure no MMIO register space access when CU is released.
-	    uint32_t *p = mCuMaps[ipIndex].first;
+	    uint32_t *p = mCuMaps[ipIndex].addr;
 	    if (p) {
-		(void) munmap(p, mCuMaps[ipIndex].second);
-		mCuMaps[ipIndex] = std::make_pair<uint32_t*, uint32_t>(nullptr, 0);
+		(void) munmap(p, mCuMaps[ipIndex].size);
```

Comment:
It it safe to ignore non-zero return value?
If yes, then let's now use 
`std::ignore = munmap(...)`

Reply:
It was ignore from day one. This is safe to ignore. Will use std::ignore